torch/distributed/tensor/examples/visualize_sharding_example.py

```diff
@@ -1,81 +1,127 @@
 """
 To run the example, use the following command:
-torchrun --standalone --nnodes=1 --nproc-per-node=4 visualize_sharding_example.py
+TERM=xterm-256color torchrun --nproc-per-node=4 visualize_sharding_example.py
```

Comment:
The environment variable XTERM controls the terminal's coloring capability. Change the default value to `xterm-256color` to release the power of your terminal app.